pinot-core/src/test/java/org/apache/pinot/core/operator/combine/CombineSlowOperatorsTest.java

```diff
@@ -65,7 +65,8 @@ public void setUp() {
   public void testSelectionOnlyCombineOperator() {
     List<Operator> operators = getOperators();
     SelectionOnlyCombineOperator combineOperator = new SelectionOnlyCombineOperator(operators,
-        QueryContextConverterUtils.getQueryContextFromPQL("SELECT * FROM table"), _executorService, TIMEOUT_MS);
+        QueryContextConverterUtils.getQueryContextFromPQL("SELECT * FROM table"),
```

Comment:
This change doesn't seem to be related to the PR

Reply:
done

Reply:
I meant this change can be undone right?

Reply:
Oh, this is performed by IntellJ reformat during rebase. Since it's a common practice to limit the length of line and this PR already get merged, I vote to not undo this change.